utilities/persistent_cache/persistent_cache_test.cc

```diff
@@ -29,6 +29,12 @@ static void OnOpenForRead(void* arg) {
       std::bind(OnOpenForRead, std::placeholders::_1));
 }
 
+static void OnDeleteDir(void* arg) {
+  char* dir = static_cast<char*>(arg);
+  std::string cmd("rm -r -f " + std::string(dir));
```

Comment:
This is not platform independent. Please do it using Env. List all the files and delete them, as what we do in some other tests.